packages/block/src/header.ts

```diff
@@ -1000,12 +860,12 @@ export class BlockHeader {
   }
 
   /**
-   * Internal helper function to create an annotated error message
+   * Helper function to create an annotated error message
    *
    * @param msg Base error message
    * @hidden
    */
-  protected _errorMsg(msg: string) {
+  private _errorMsg(msg: string) {
```

Comment:
Small nit here, this should remain `protected` to allow for sub-classing.

Reply:
Got it.  Adding this to #1982